core/src/main/java/com/sequenceiq/cloudbreak/service/stack/connector/DiskAttachUtils.java

```diff
@@ -2,18 +2,29 @@
 
 public final class DiskAttachUtils {
```

Comment:
The class could be renamed to VolumeAttachUtils, if all methods have been renamed